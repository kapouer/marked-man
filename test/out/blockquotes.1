T(1)                                                                      T(1)

NNAAMMEE
       tt - test

tteesstt
         A blockquote here.
         Continuing on second line.

       Hello.

                                 January 1979                             T(1)
