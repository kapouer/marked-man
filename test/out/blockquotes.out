[4mT[24m(1)                        General Commands Manual                       [4mT[24m(1)

[1mNAME[0m
       [1mt [22m- test

[1mtest[0m
         A blockquote here.
         Continuing on second line.

       Hello.

              Another  block  quote. With a long title. That will break a line
              eventually later.
              that continues on second line
              continuation

                                 January 1979                             [4mT[24m(1)
